examples/airflow/shared_state/sky_k8s_example_xcoms.py

```diff
@@ -0,0 +1,81 @@
+# This is a WIP example that uses xcom serialization to pass state.db and sky keys between tasks.
```

Comment:
This file can be removed, but keeping it for now as a useful reference for anyone considering xcoms for transferring state.

Reply:
I would either:
1. add a sentence in README linking to this file; or
2. remove this file

Otherwise, it might be confusing to see a "dangling" file here ; )